pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBase.java

```diff
@@ -929,6 +934,9 @@ protected void callMessageListener(Message<T> msg) {
         } catch (Throwable t) {
             log.error("[{}][{}] Message listener error in processing message: {}", topic, subscription,
                     msg.getMessageId(), t);
+        } finally {
+            executorQueueSize.decrementAndGet();
+            triggerListener();
```

Comment:
Maybe stackOverflow.

Reply:
Why could occur this?

Reply:
Oh, I made a mistake, they are in different methods